Jenkinsfile

```diff
@@ -0,0 +1,47 @@
+pipeline {
+  agent { label "android" }
+
+  stages {
+    stage("File Setup") {
+      steps {
+        sh "mv .jenkins/ci-local.properties local.properties"
+        sh "docker create -t -i -v $WORKSPACE:/opt/project-android --name jd-container simplymadeapps/docker-android:1.0.0"
+        sh "docker start jd-container"
+      }
+    }
+    
+    //We are running as a circleci user and not as root so we have to add sudo to everything
+    
```

Comment:
Add linting step.

Reply:
Done.
I had 2 warnings when I ran my lint command.
1) It said I was creating a `SharedPreferences.Editor` object but never calling `commit()` on it.  It does not realize that the `put()` methods are responsible for calling `commit()`.  I am suppressing this one because I know I am doing the right thing it just isn't smart enough to know.
2) It told me there was a newer version of the `com.android.support:support-annotations` I should use.  This was just changing from 27.1.1 to 28.0.0.

Now all lint checks pass (including warnings) and the build will fail if this ever changes.  I do not need a baseline file because everything is already satisfied.